samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java

```diff
@@ -916,7 +916,7 @@ else if (!qbean.getQuestionType().equals(TypeIfc.MATCHING.toString())) // matchi
       getImageMapQuestionScores(publishedItemTextHash, publishedAnswerHash, (List) scores, qbean, (List) text);
     }
 
-    long attemptCount = Optional.ofNullable(qbean.getN()).map(Long::valueOf).orElse(0L);
+    long attemptCount = Optional.ofNullable(qbean.getNumResponses()).map(Long::valueOf).orElse(0L);
```

Comment:
@mpellicer Don't laugh!

Reply:
Nahh, laughing is optional.

Optional.ofNullable(null).orElse("hahaha I told you")

Reply:
if (true) {
return "You are just pretending its optional!";
}